kuma/users/views.py

```diff
@@ -935,3 +935,37 @@ def stripe_hooks(request):
         )
 
     return HttpResponse()
+
+
+def _send_payment_received_email(payment_intent, locale):
+    user = get_user_model().objects.get(stripe_customer_id=payment_intent.customer)
+    subscription_info = _retrieve_and_synchronize_subscription_info(user)
+    locale = locale or settings.WIKI_DEFAULT_LANGUAGE
+    context = {
+        "payment_date": datetime.fromtimestamp(payment_intent.created),
+        "next_payment_date": subscription_info["next_payment_at"],
+        "invoice_number": payment_intent.id,
```

Comment:
This is incorrect. That ID is a Stripe ID. The correct Invoice number is this:
<img width="578" alt="Screen Shot 2020-04-03 at 9 56 30 AM" src="https://user-images.githubusercontent.com/26739/78368304-6dbc8300-7591-11ea-977c-48a46e848a3f.png">

You can see it on the PDF too:
<img width="279" alt="Screen Shot 2020-04-03 at 9 57 05 AM" src="https://user-images.githubusercontent.com/26739/78368346-7e6cf900-7591-11ea-8aad-7f505ff03f22.png">